x-pack/legacy/plugins/maps/public/connected_components/layer_panel/layer_settings/_layer_settings.scss

```diff
@@ -0,0 +1,6 @@
+.mapLayerSettings__globalFilterRow {
+    .euiFormRow__fieldWrapper {
```

Comment:
We usually try to stay away from targeting `.eui` classes in conumsing applications. What was the problem here you were trying to solve?

Reply:
When a `<EuiSwitch compressed />` is inside a `<EuiFormRow display="columnCompressed" />` it doesn't vertically center.

<img width="414" alt="Screenshot 2019-09-26 at 15 10 16" src="https://user-images.githubusercontent.com/2750668/65695696-eafdbd00-e06f-11e9-8d90-62a89ec1aa1e.png">

Reply:
Ahh can you try doing `<EuiFormRow display="columnCompressedSwitch" />`

Reply:
Done!